x-pack/plugins/monitoring/public/components/elasticsearch/nodes/cells.js

```diff
@@ -28,40 +50,72 @@ const metricVal = (metric, format, isPercent, units) => {
   return formatMetric(metric, format, units);
 };
 
-const noWrapStyle = { overflowX: 'hidden', whiteSpace: 'nowrap' };
-
 function MetricCell({ isOnline, metric = {}, isPercent, ...props }) {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const onButtonClick = () => setIsPopoverOpen((isPopoverOpen) => !isPopoverOpen);
+  const closePopover = () => setIsPopoverOpen(false);
+
   if (isOnline) {
     const { lastVal, maxVal, minVal, slope } = get(metric, 'summary', {});
     const format = get(metric, 'metric.format');
     const units = get(metric, 'metric.units');
 
+    const tooltipItems = [
+      {
+        title: i18n.translate('xpack.monitoring.elasticsearch.node.cells.tooltip.trending', {
+          defaultMessage: 'Trending',
+        }),
+        description: getDirection(slope),
+      },
+      {
+        title: i18n.translate('xpack.monitoring.elasticsearch.node.cells.tooltip.max', {
+          defaultMessage: 'Max value',
+        }),
+        description: metricVal(maxVal, format, isPercent, units),
+      },
+      {
+        title: i18n.translate('xpack.monitoring.elasticsearch.node.cells.tooltip.min', {
+          defaultMessage: 'Min value',
+        }),
+        description: metricVal(minVal, format, isPercent, units),
+      },
+    ];
+
+    const button = (
+      <EuiIcon
+        onClick={onButtonClick}
+        tabIndex="0"
+        type={getIcon(slope)}
+        data-test-subj={`monitoringCellIcon-${props['data-test-subj']}`}
+        title={i18n.translate('xpack.monitoring.elasticsearch.node.cells.tooltip.iconLabel', {
```

Comment:
I suggest having the icon in front of the number, I think this way it is easier to read.

Reply:
K, will do